### PR TITLE
create guide to ICE-breaker groups and specifically advice for LLVM

### PR DESCRIPTION
**Pull request by nikomatsakis**
r? @spastorino

### CODE REVIEW

File: src/ice-breaker/about.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+- By **isolated**, we mean that we do not expect large-scale refactoring
+  to be required to fix the bug.
+- By **middle priority**, we mean that we'd like to see the bug fixed,
+  but it's not such a burning problem that we are dropping everything
+  else to fix it. The danger with such bugs, of course, is that they
+  can accumulate over time, and the role of the ICE-breaker groups is
+  to try and stop that from happening!
+
+<a name="join"></a>
```
Comment (spastorino) [inline src/ice-breaker/about.md]: What was the idea with this?

File: src/ice-breaker/llvm.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+The ["Debugging LLVM"][d] section of the
+rustc-guide gives a step-by-step process for how to help debug bugs
+caused by LLVM. In particular, it discusses how to emit LLVM IR, run
+the LLVM IR optimization pipeliness, and so forth. You may also find
+it useful to look at the various codegen options listed under `-Chelp`
+and the internal options under `-Zhelp` -- there are a number that
+pertain to LLVM (just search for LLVM).
+
+[d]: ../codegen/debugging.md
```
Comment (spastorino) [inline src/ice-breaker/llvm.md]: did you use just d for some particular reason? :)

Comment (nikomatsakis) [reply src/ice-breaker/llvm.md]: it's short?

File: src/ice-breaker/about.md (hunk 1 above)
Comment (nikomatsakis) [reply src/ice-breaker/about.md]: that we are going to want to link to this page, I think

Comment (spastorino) [reply src/ice-breaker/about.md]: I'm questioning mainly because I wanted to know if you wanted to merge this or was WIP.

Comment (nikomatsakis) [reply src/ice-breaker/about.md]: in particular I wanted a "stable" anchor in case we change the section heading later

**Pull request merged**